ci-operator/step-registry/assisted/common/setup/prepare/assisted-common-setup-prepare-commands.sh

```diff
@@ -96,7 +96,9 @@ export PULL_SECRET=\$(cat /root/prod/pull-secret)
 export OFFLINE_TOKEN=\$(cat /root/prod/offline-token)
 export REMOTE_SERVICE_URL=https://api.openshift.com
 export NO_MINIKUBE=true
-export MAKEFILE_TARGET='setup test_parallel'
+export MAKEFILE_SETUP_TARGET=setup
+export MAKEFILE_TARGET=test_parallel
+export WORKER_DISK={{ '120 GB' | human_to_bytes }}
```

Comment:
I just replicated what was done in the old script but I don;t know if it's still required

Reply:
test-infra should have handled it: https://github.com/openshift/assisted-test-infra/blob/c8c6bdbf2b2c3be1d4ee6fc1bcd96dcb466d30af/src/triggers/default_triggers.py#L12

Reply:
let's not set it then ?

Reply:
yes let's try without setting it